tests/mentorship_relation/test_api_accept_request.py

```diff
@@ -47,6 +47,101 @@ def test_accept_mentorship_request(self):
             self.assertDictEqual(messages.MENTORSHIP_RELATION_WAS_ACCEPTED_SUCCESSFULLY,
                              json.loads(response.data))
 
+    # Assuming User1 sent request X to User2, User2 accepts this request, while
+    # User2 is NOT involved in a current mentorship relation
+    # 200, MENTORSHIP_RELATION_WAS_ACCEPTED_SUCCESSFULLY response
+    def test_free_recepient_accept_request(self):
```

Comment:
Can you remove this test since it is already in place?

Reply:
done